README.md

```diff
@@ -1403,6 +1403,31 @@ However, when we try to call `makeSound()` on an instance of the Animal class `(
 
 ---
 
+###### 42. Default Import Export (Level: Medium)
+
+In JavaScript, when importing a default export from a module, which syntax correctly assigns an alias "myAlias" to the default import?
+
+- A: import { myAlias as default } from 'module';
+- B: import default as myAlias from 'module';
+- C: import myAlias as default from 'module';
+- D: import { default as myAlias } from 'module';
```

Comment:
The question is good but I think we can improve the MCQ options to show both possible ways of adding alias with default imports:

```diff
- - A: import { myAlias as default } from 'module';
- - B: import default as myAlias from 'module';
- - C: import myAlias as default from 'module';
- - D: import { default as myAlias } from 'module';

+ - A: import default as myAlias from 'module';
+ - B: import myAlias from 'module';
+ - C: import { default as myAlias } from 'module';
+ - D: Both option B and C
```